src/presenters/twitter/card-presenter.php

```diff
@@ -26,10 +26,13 @@ public function get() {
 		/**
 		 * Filter: 'wpseo_twitter_card_type' - Allow changing the Twitter card type.
 		 *
-		 * @api string $card_type The card type.
+		 * @deprecated 19.8
 		 *
+		 * @param string $card_type The card type.
 		 * @param Indexable_Presentation $presentation The presentation of an indexable.
 		 */
-		return \trim( \apply_filters( 'wpseo_twitter_card_type', $this->presentation->twitter_card, $this->presentation ) );
+		\apply_filters_deprecated( 'wpseo_twitter_card_type', [ $this->presentation->twitter_card, $this->presentation ], 'WPSEO 19.8', '', 'The only supported value is summary_large_image' );
```

Comment:
@enricobattocchi Aside from the above, the rest LGTM and also acceptance is 👍 If you agree with the recommendation, I can push that small change and then merge.

I also added a small part in the Test Instructions that check the deprecation of the filter.

Final thing, if we merge this, I think we need to at least give a heads up to the Settings UI team, because right now the only that "downgrades" the setting in the db is the lack of a hidden field in that form and if that field is re-instated with the new Settings, we won't be downgrading that setting - not that it's a crucial thing, but just to make sure we're all on the same page.